logstash-core/src/test/java/org/logstash/StringInterpolationTest.java

```diff
@@ -149,6 +151,24 @@ public void TestValueIsHash() throws IOException {
         assertEquals("{\"k1\":\"v\"}", StringInterpolation.evaluate(event, path));
     }
 
+    @Test
+    public void TestTimeNow() throws IOException {
```

Comment:
I know that the other test methods starts with capital letter, but in Java methods starts with lowercase.
I would also suggest to rename:
- `TestTimeNow` -> `testGivenAnEventWithTimeNowSubstitutionVariableWhenEvaluatedReplaceWithCurrentTimestamp`
- `TestBadTimeNow` -> `testGivenAnEventWithASubstitutionVariableThatContainsTheTimeNowMarkerWhenEvaluatedNoReplacementWithCurrentTimestampHappen`